tests/bake.go

```diff
@@ -1274,3 +1275,87 @@ target "default" {}
 	require.Empty(t, md.Default.BuildRef)
 	require.Len(t, md.Default.ResultJSON.Warnings, 3)
 }
+
+func testBakeCheckCallOutput(t *testing.T, sb integration.Sandbox) {
+	t.Run("check for warning count msg in check without warnings", func(t *testing.T) {
+		dockerfile := []byte(`
+FROM busybox
+COPY Dockerfile .
+		`)
+		bakefile := []byte(`
+	target "default" {}
+	`)
+		dir := tmpdir(
+			t,
+			fstest.CreateFile("docker-bake.hcl", bakefile, 0600),
+			fstest.CreateFile("Dockerfile", dockerfile, 0600),
+		)
+
+		cmd := buildxCmd(
+			sb,
+			withDir(dir),
+			withArgs("bake", "--call", "check", "--metadata-file", filepath.Join(dir, "md.json")),
```

Comment:
Why is the `--metadata-file` in these checks? Seems to be unused?

Reply:
Removed!